core/src/main/scala/tech/beshu/ror/es/LogBuildInfoMessage.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.es
+
+object LogBuildInfoMessage extends org.apache.logging.log4j.scala.Logging{
```

Comment:
maybe instead of object, it can be trait so we could mix it in. And the code could be invoked in constructor. 

If there is no possible to create buildInfo, proper message should be logged and code should be able to go on.

Reply:
I made trait, but I'm not sure if this is best approach. I don't like shearing logic by mixins. Additionally I think we should always provide buildInfo. If there is no possible to create buildInfo, we should fail, and fix it.

Reply:
I see two approaches here:
* treat build info as a log - if there is no such one, let it fall, log and go on
* treat build info as always available one - here we should have it tested, to be always sure that our plugin won't crash because of such not important thing like version logging

Reply:
I've chosen second one. Build info depends on properties file provided in resources. If this file or one of versions value's wouldn't be available, logging message will fail, crashing plugin. This properties file is provided on package, and it's correctness is tested in every test running plugin. I know it should be tested directly, but I would have to simulate corrupted build.

Reply:
right - our integration tests should be able to catch the situation when properties file is malformed/not present.